libkbfs/block_journal.go

```diff
@@ -62,6 +62,14 @@ type blockJournal struct {
 	j          diskJournal
 	refs       map[BlockID]blockRefMap
 	isShutdown bool
+
+	// Tracks the total size of on-disk blocks that will be put to the
+	// server (i.e., does not count reference adds).  It is only
+	// accurate for users of this journal that properly flush entries;
+	// in particular, direct calls to `removeReferences` can cause
```

Comment:
"direct calls to removeReferences" -> "calls to removeReferences with removeUnreferencedBlocks set to true", right?

Although I guess my pending PR will change this slightly, but I can fix up the comment when you land this.